www/src/pages/components/radio/ios/index.mdx

```diff
@@ -3,31 +3,119 @@ title: Radio
 description: Control for selecting an option inside a form
```

Comment:
nit: Could you also change this description to "Boxes for checking and unchecking single values in forms." for consistency with the other platforms while you're here?

Reply:
I'm not sure that's the best description we could have but I got it in for consistency.